models/message.js

```diff
@@ -61,4 +61,10 @@ messageSchema.post('save', async function (doc) {
   await Chat.updateOne({ _id: chat }, { latestMessage: _id })
 })
 
+messageSchema.post('deleteMany', async function (doc) {
+  const { chat } = doc
+
+  await Chat.updateOne({ _id: chat }, { latestMessage: null })

```

Comment:
i think there is no need of destructuring object `doc` if we use here only one value, it would be better like that:
```javascript
{ _id: doc.chat }
```